pkg/porter/explain.go

```diff
@@ -20,14 +21,15 @@ type ExplainOpts struct {
 
 // PrintableBundle holds a subset of pertinent values to be explained from a bundle.Bundle
 type PrintableBundle struct {
-	Name         string                `json:"name" yaml:"name"`
-	Description  string                `json:"description,omitempty" yaml:"description,omitempty"`
-	Version      string                `json:"version" yaml:"version"`
-	Parameters   []PrintableParameter  `json:"parameters,omitempty" yaml:"parameters,omitempty"`
-	Credentials  []PrintableCredential `json:"credentials,omitempty" yaml:"credentials,omitempty"`
-	Outputs      []PrintableOutput     `json:"outputs,omitempty" yaml:"outputs,omitempty"`
-	Actions      []PrintableAction     `json:"customActions,omitempty" yaml:"customActions,omitempty"`
-	Dependencies []PrintableDependency `json:"dependencies,omitempty" yaml:"dependencies,omitempty"`
+	Name          string                `json:"name" yaml:"name"`
+	Description   string                `json:"description,omitempty" yaml:"description,omitempty"`
+	Version       string                `json:"version" yaml:"version"`
+	PorterVersion string                `json:"porter_version" yaml:"porter_version"`
```

Comment:
Can you change this to use camel case (`porterVersion`) instead of snake case (`porter_version`)? That's what we are using elsewhere when writing to json.